.tekton/pull-request.yaml

```diff
@@ -113,7 +113,7 @@ spec:
               type: string
           steps:
             - name: e2e-test
-              image: quay.io/redhat-appstudio/e2e-tests:main
+              image: quay.io/psturc/e2e-tests:fix-checking-pipelines-running-in-parallel
```

Comment:
Should this commit be included in this PR?

Reply:
removed